CheapShopWeb/Controllers/ProductController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using CheapShopWeb.DataContext;
+
+namespace CheapShopWeb.Controllers
+{
+    public class ProductController : Controller
+    {
+        // GET: Product
+        private ProductDbContext _db;
+
+        public ProductController()
+        {
+            _db = new ProductDbContext();
+        }
+
+        public ActionResult Product()
+        {
+            var products = _db.Products.ToArray();
+            return View(products);
+        }
+    }
+}
```

Comment:
Ką daro šis controller?

Reply:
Šiuo metu tai tiesiog partraukia duomenis is duombazes ir nusiuncia i view'ą